newsletter/v208.0.1-February-2022.md

```diff
@@ -0,0 +1,118 @@
+# February 2022 StyleGuide newsletter
+
+Combined update notes, summarising changes in the style guide from a recent month. This is the first edition of the newsletter, so we decided to include features from a longer period. Some of the key highlights include:
+
+- [Components accessibility improvements](#accessibility-improvements)
+- [Components color API changes](#components-color-api-changes) - caused by introduction of new color tokens.
+- [Dialog component release](#dialog-component)
+- [Responsive props](#responsive-props) - added responsive props support to Flex and text components.
+- [UX motion tokens](#ux-motion)
+- [Elevation in the UI](#elevation-in-the-ui)
+
+## Accessibility improvements
+
+- Rating
+  - Star uses radio input and has accessible label
+  - Rating has accessible descriptions for current rate value and rate action
+- HomeButton
+  - `aria-*` props renamed to use aria naming convention
+  - `aria-label` added for a link (defaults to `type`)
+  - `altTag` renamed to `alt`
+- Logo
+  - `alt` defaults to `type`
+- FlashMessage
+  - `alert` role added
+- Avatar
+  - focus style is visible
+  - new props: `alt` & `ariaLinkLabel`
+- Icon
+  - added accessible `title` & `description` props for `Icon`
+  - added `role` to `svg` in `Icon` (default: `"img"`)
+  - adjusted colors for high contrast mode (only Windows)
+- AccordionItem
+  - added role `heading` and prop `headingLevel`
+
+Detailed accessibility documentation for each component in [Storybook](https://style-guide.brainly.com/) is coming soon.
+
+## Components color API changes
+
+Color API changes caused by introduction of new [color tokens](https://style-guide.brainly.com/?path=/story/foundation-%E2%9C%A8-colors--page).
+
+- Icon
+- MobileIcon
+- SubjectIcon
+- MathSymbol
+- IconAsButton
+- Text
+- TextBit
+- Link
+- Headline
+- Bubble
+- Box
+- Label
+- CardHole
+- Spinner, SpinnerContainer
+- Counter
+- Overlay
+- FileHandler
+- Media
+- Separators
+
+There's only `Button` component API left to be refactored, though the component itself is already consuming new color tokens.
+
+## New icons
+
+There was a bunch of new icons added in recent releases:
+
+- mobile
+  - circle_fill
```

Comment:
It would be nice to have some screenshots of icons (or in general when we have some simple visual changes). I can help you make those.